docs/docs/1.0.6/documentation/health.md

```diff
@@ -0,0 +1,160 @@
+# Health checks
+
+Health is an extremely important part of the microservice architecture for various reasons. Load balancers rely on the 
+health status of the application while picking endpoint from load balancer set. Users want the application state to be 
+observable through the GUI or CLI. Orchestrators like Kubernetes also want to know the application status to manage 
+lifecycle of the application. 
+
+Kuma supports several aspects of the health checking. There are two policies which allows configuring active and passive:
+[Health Check](../policies/health-check.md) and [Circuit Breaker](../policies/circuit-breaker.md).
```

Comment:
`... to check the connection between services`

Reply:
Sorry, I don't get that, do you want it to look like:

Kuma supports several aspects of the health checking. There are two policies which allows configuring active and passive health checks:
[Health Check](../policies/health-check.md) and [Circuit Breaker](../policies/circuit-breaker.md) to check the connection between services.